components/card/stories/card.stories.js

```diff
@@ -23,11 +23,15 @@ export default {
 			},
 			control: { type: "file", accept: ".svg,.png,.jpg,.jpeg,.webc" },
 		},
-		isQuiet,
+		isQuiet: {
+			...isQuiet,
+			description: "`showAsset: 'image'` must be selected to properly render the quiet styles.",
```

Comment:
If this is a requirement, should we update the controls with an if that requires it?

Reply:
I took another look at this. I don't think `showAsset` is actually a requirement, so I'm handling the `isQuiet` arg differently in the template now. Let me know what you think! https://github.com/adobe/spectrum-css/pull/3443/commits/e2a3bb8ed9c9b7b35a0a11c11df97c8d86d56899